packages/itinerary-body/src/place-row.js

```diff
@@ -0,0 +1,291 @@
+        <Styled.PlaceDetails>
+          {/* Dot separating interlined segments, if applicable */}
+          <Styled.PlaceHeader>
+            {/*
+              TODO: Need to rework this -- Need to display a marker
+              for an interline place
+            */}
+            {interline && <Styled.InterlineDot>&bull;</Styled.InterlineDot>}
+            <Styled.PlaceName>
+              {interline ? (
+                <>
+                  Stay on Board at <b>{place.name}</b>
+                </>
+              ) : (
+                <>{getPlaceName(place, config.companies)}</>
+              )}
+              {/* TODO: take another pass on this when working the Transit Leg */}
+              {/* Place subheading: Transit stop */}
+              {place.stopId && !interline && (
+                <Styled.StopIdSpan>
+                  ID {place.stopId.split(":")[1]}
+                </Styled.StopIdSpan>
+                /*
+                TODO: There is no explicit stop button on the mocks.
+                Have a question out to marketing as to whether the above StopID
+                is a button to navigate the user to the arrival list for the stop
+                Thats what the button below does
+              */
+                /* <ViewStopButton stopId={place.stopId} /> */
+              )}
+            </Styled.PlaceName>
+          </Styled.PlaceHeader>
+
+          {/* Place subheading: rented vehicle (e.g., scooter, bike, car) pickup */}
+          {leg && (leg.rentedVehicle || leg.rentedBike || leg.rentedCar) && (
+            <RentedVehicleLeg config={config} leg={leg} />
+          )}
+          {/* Show the leg, if present */}
+          {leg &&
+            (leg.transitLeg ? (
+              /* This is a transit leg */
+              <TransitLegBody
+                leg={leg}
+                legIndex={legIndex}
+                setActiveLeg={setActiveLeg}
+                longDateFormat={longDateFormat}
+                operator={
+                  leg.agencyId && getOperatorFromConfig(leg.agencyId, config)
+                }
+                setViewedTrip={setViewedTrip}
+                timeFormat={timeFormat}
+              />
+            ) : (
+              /* This is an access (e.g. walk/bike/etc.) leg */
+              <AccessLegBody
+                config={config}
+                diagramVisible={diagramVisible}
+                followsTransit={followsTransit}
+                leg={leg}
+                LegIcon={LegIcon}
+                legIndex={legIndex}
+                routingType={routingType}
+                setActiveLeg={setActiveLeg}
+                setLegDiagram={setLegDiagram}
+                showElevationProfile={showElevationProfile}
+                timeOptions={timeOptions}
+              />
+            ))}
+        </Styled.PlaceDetails>
+      </Styled.DetailsColumn>
+      <Styled.MapButtonColumn hideBorder={hideBorder.toString()}>
```

Comment:
Is this 
![image](https://user-images.githubusercontent.com/56846598/71018306-431fea80-20c6-11ea-872d-a331faddf159.png)
 supposed to reproduce this portion 
![image](https://user-images.githubusercontent.com/56846598/71018248-2c799380-20c6-11ea-901a-92d7162504a5.png)
 of the UI?

Reply:
The trimet.org mockup moved some things around. I am not exactly sure what the intent is with putting the StopID and kind of omitting the place sub-header is some areas, but I left it as is. I assume this will need more iteration to get it looked a little nicer.

Reply:
I don't quite know what it is supposed to do. It just accepts a function and whatever is done as a result of calling that function is out of the scope of this component.